backend/app/views/spree/admin/promotion_categories/index.html.erb

```diff
@@ -27,8 +29,12 @@
         <td><%= promotion_category.name %></td>
         <td><%= promotion_category.code%></td>
         <td class="actions">
-          <%= link_to_edit promotion_category, :no_text => true %>
-          <%= link_to_delete promotion_category, :no_text => true %>
+          <% if can?(:edit, promotion_category) %>
```

Comment:
I was confused that we use `:edit` here and `:update` below. Turns out those are aliases but consistency is nice so I'll nitpick here.